synapse/lib/iq.py

```diff
@@ -827,6 +833,12 @@ def eq(self, x, y):
         '''
         self.assertEqual(x, y)
 
+    def eqish(self, x, y):
```

Comment:
I would probably add a `places` kwarg instead of hardcoding 6.

Reply:
ok